service/profile.go

```diff
@@ -0,0 +1,54 @@
+package service
+
+import (
+	"context"
+
+	"github.com/compliance-framework/configuration-service/domain"
+	"github.com/compliance-framework/configuration-service/event"
+	mongoStore "github.com/compliance-framework/configuration-service/store/mongo"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+//todo
+//create an endpoint to save a profile (only with its title for now),
+// get a list of profiles
+
+type ProfileService struct {
+	profileCollection *mongo.Collection
+	publisher         event.Publisher
```

Comment:
We don't need a publisher as crud operations on a profile won't trigger events on NATS.